<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="firefoxtest"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="0.058">
  <properties />
            <testcase name="tc_1" time="0.040" classname="com.durgasoft.parallelexecution.Test1">
                <failure type="java.lang.AssertionError"
                  message="expected [test123] but found [test]"
              >
<![CDATA[java.lang.AssertionError: expected [test123] but found [test]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:496)
  org.testng.Assert.assertEquals(Assert.java:125)
  org.testng.Assert.assertEquals(Assert.java:178)
  org.testng.Assert.assertEquals(Assert.java:188)
  com.durgasoft.parallelexecution.Test1.tc_1(Test1.java:13)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
  org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  java.lang.Thread.run(Thread.java:748)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="tc_2" time="0.013" classname="com.durgasoft.parallelexecution.Test1" />
                <testcase name="tc_3" time="0.005" classname="com.durgasoft.parallelexecution.Test1" />
      
</testsuite>
